### PR TITLE
Update sketchfab viewer aspect ratio

### LINKED ISSUE #888: Constrain Sketchfab Model height on single result view
## Description

Sketchfab models are really tall on the new single result view.

https://search-staging.openverse.engineering/image/00a6a7a4-6605-4cba-bacf-a08b0da546b9

CleanShot 2022-02-16 at 14 55 41@2x

How could we constrain these so the entire model is visible at once?

CC @panchovm for design thought.

### PR DESCRIPTION
**Pull request by sarayourfriend**
## Fixes

Fixes #888 by @zackkrida 

## Description

Constrains the sketchfab viewer to the 1000px/500px (2/1) aspect ratio indicated by @panchovm in the comments in the linked issue.

## Testing Instructions

Check out this branch and run the app using `pnpm dev`. Then visit `localhost:8443/image/00a6a7a4-6605-4cba-bacf-a08b0da546b9` and verify that the viewer is the correct aspect ratio at various screen widths.

I thought about adding a snapshot test for this but wasn't sure about it. Would we mock the request to the sketchfab-viewer script and the subsequent sketchfab API requests for the model information? Seems thorny!

## Checklist

- [x] My pull request has a descriptive title (not a vague title like `Update index.md`).
- [x] My pull request targets the *default* branch of the repository (`main`) or a parent feature branch.
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [N/A] I added or updated tests for the changes I made (if applicable).
- [N/A] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no visible errors.

[best_practices]:https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

File: src/components/VSketchFabViewer.vue [hunk 1]
```diff
@@ -0,0 +1,82 @@
+<template>
+  <div class="block max-w-[1000px] max-h-[500px] aspect-[2/1]">
+    <iframe
+      id="sketchfab-iframe"
+      ref="node"
+      src=""
+      sandbox="allow-scripts allow-same-origin allow-popups allow-forms"
+      title="Sketchfab iframe"
```
Comment (obulat) [inline src/components/VSketchFabViewer.vue:8]: Can we use i18n here? I've read today that there's inconsistency in the a11y tools, so it is advised to use both `title` and `aria-label` for iframes.

**Review by obulat — APPROVED** [review]
Looks well both on a large screen and on mobile.
The iframe title/ `aria-label` can be fixed here or in another PR.

File: src/components/VSketchFabViewer.vue (hunk 1 above)
Comment (sarayourfriend) [reply src/components/VSketchFabViewer.vue:8]: Oops, yes, definitely! I will make that change.

**Comment by panchovm** [comment]
It looks great to me. Nice work

**Review by zackkrida — APPROVED** [review]
Looks great!

**Pull request merged**